### ISSUE TITLE
Send Reply issue

### ISSUE THREAD

**Issue by malkeet2018** [nodemailer/wildduck#60]
Hi,
I am sending and receiving messages with api as mentioned
curl -i -XPOST "http://localhost:8080/users/59fc66a03e54454869460e45/submit" \
-H 'Content-type: application/json' \
-d '{
  "to": [{
    "address": "andris@ethereal.email"
  }],
  "subject": "Hello world!",
  "text": "Test message"
}'

But I need to send reply so that recepient can see my reply in his Inbox 
{
  "reference": {
    "mailbox": "59fc66a03e54454869460e47",
    "id": 15,
    "action": "replyAll"
  },
  "text": "Yeah, sure"
}
is not working , what should be complete payload for it ?
please help

**Comment by sankath** [comment]
CURL response for reply call is:

```
{
  "success": true,
  "message": {
    "id": 173,
    "mailbox": "5a79e27b0f62ca206364548b",
    "queueId": false
  }
}
```

**Comment by sankath** [comment]
Seems like reply message is getting lost because queueId is being returned false in response.

**Comment by hamedbaatour** [comment]
### Solution
For replies to be sent correctly you should add `to` object in your request alongside the `reference` object.
just add the receivers email addresses and names in the `to` array, like this:
`to: [ {name: 'andris' , address: 'andris@ethereal.email'} ]`
the `reference` object is there to indicate that this is a reply (subject + headers) but doesn't set who is going to receive the reply email.
I agree that `replyToAll` option logically should prefill the `to` array but it seems like the API works this way currently.

closes #60

@andris9
